chainer/distributions/independent.py

```diff
@@ -236,8 +236,8 @@ def _kl_independent_independent(dist1, dist2):
     # KL(p || q) and do a `reduce_sum` on the reinterpreted batch dimensions.
     if dist1.event_shape == dist2.event_shape:
         if p.event_shape == q.event_shape:
-            num_reduce_dims = len(dist1.event_shape) - len(p.event_shape)
-            reduce_dims = tuple([-i - 1 for i in range(0, num_reduce_dims)])
+            n_reduce_dims = len(dist1.event_shape) - len(p.event_shape)
```

Comment:
I wonder if `reduce_dim` is better.
I want to ask @Crissman's opinion. :pray:

Reply:
`n_reduce_dim` looks better to me.

Reply:
Updated. (Sorry, I missed your comment. 🙇)